### DISCUSSION TITLE
simultaneous estimation of common parameter for 2 compounds

### DISCUSSION THREAD

**Discussion by estelley** [Open-Systems-Pharmacology/Forum#1231] [Questions & Problems] [answered]
Hello,
I wonder if it is possible in MoBi to have 2 compounds simultaneously in a same PBPK model in order to estimate a common parameter (e.g. reference concentration of enzyme or transporter) especially when they have same mechanistic processes involved?
If it is possible, how can we do it?
Thanks

**Comment by msevestre** [comment, marked as answer]
see https://github.com/Open-Systems-Pharmacology/Forum/discussions/1227
